src/backend/distributed/shared_library_init.c

```diff
@@ -154,7 +154,11 @@ _PG_init(void)
 	planner_hook = multi_planner;
 
 	/* register utility hook */
+#if (PG_VERSION_NUM >= 100000)
+	ProcessUtility_hook = multi_ProcessUtility10;
```

Comment:
Named it `multi_ProcessUtility9x` because it looks cool.

![c60](https://user-images.githubusercontent.com/50673/27050480-ed81de1e-4f6f-11e7-9f00-ac6b0a118801.png)